api/db/models/petition.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Enum
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql import func
+from ..base_class import Base
+from .supporter_petitions import supporter_petitions
+import enum
+
+
+class Status(enum.Enum):
+    active = "active"
+    victory = "victory"
+    closed = "closed"
+
+
+class Petition(Base):
+    __tablename__ = "petition"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    image = Column(String)
+    decision_maker = Column(String, nullable=False)
```

Comment:
Isn’t the decision maker also a user?

Reply:
Not always. After investigation, I found that in most cases in different petitions web services those influent persons who could do something to solve a petition problem don't have an account  on those sites and they a mentioned there as strings))

Reply:
ok, then we can’t validate this field or show available options, and it's the user's responsibility to choose the right one.